cob_teleop/ros/src/joy_init

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+#
+# Copyright 2017 Fraunhofer Institute for Manufacturing Engineering and Automation (IPA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rospy
+from sensor_msgs.msg import Joy
+from std_srvs.srv import Trigger, TriggerRequest
+
+class JoyInit(object):
+  def __init__(self):
+
+    rospy.wait_for_service("/base/driver/init")
+    self.init_client = rospy.ServiceProxy("/base/driver/init", Trigger)
+    rospy.wait_for_service("/base/driver/recover")
+    self.recover_client = rospy.ServiceProxy("/base/driver/recover", Trigger)
```

Comment:
I agree with @floweisshardt to remove the hardcoded namespace and pass the namespace via launch file...that way this can be used for other components that follow the `component/driver/service` naming convention